libraries/utils/src/render-lines-on-sp.ts

```diff
@@ -1,31 +1,27 @@
 import * as DG from 'datagrok-api/dg';
 import { Subject } from 'rxjs';
-
-export enum LineDirection {
-    pointToLowerValue = 'lower',
-    pointToHigherValue = 'higher',
-}
-
-export type ILineOpts = {
-    points: Uint32Array;
-    id: number;
-    color?: string;
-    directionCol?: string
-    direction?: LineDirection;
-    opacity?: number;
-    width?: number;
+import BitArray from './bit-array';
+
+export type ILines = {
+    from: Uint32Array;
+    to: Uint32Array;
+    colors?: string[];
+    widths?: Uint32Array;
+    opacities?: Float32Array;
+    drawArrows?: BitArray;
```

Comment:
Same, a single bool is probably enough for now.

Reply:
Done